docs/DocGenerator/config.yaml

```diff
@@ -0,0 +1,43 @@
+Project path: "../../tests/integration"
+
+Documentation path: ".."
+
+Include paths:
+  - wazuh_db:
+    path: "../../tests/integration/test_wazuh_db"
+    recursive: false
+  - vulnerability_detector:
+    path: "../../tests/integration/test_vulnerability_detector"
+
+Include regex:
+  - "^test_.*py$|readme.md$"
```

Comment:
What about something like this:
```
Include regex: "^test_.*py$|readme.md$"
```

Reply:
It is contemplated to have more than one regex here.